setup.py

```diff
@@ -18,22 +18,36 @@
 with open("requirements.txt") as requirements_file:
     requirements = requirements_file.read()
 
+
+def package_files(directory):
+    # source: https://stackoverflow.com/a/36693250
+    paths = []
+    for (path, directories, filenames) in os.walk(directory):
+        for filename in filenames:
+            paths.append(os.path.join(path, filename))
+    return paths
```

Comment:
This function should return the list relative to the package directory, i,.e., relative to `src/exam_generator`.